### PR TITLE
[DONE] horizontal stack in hectares

### PR DESCRIPTION
**Pull request by RoelvandenBerg**

### CODE REVIEW

**Comment by ernstkui** [comment]
looks good, i will check it out in a little bit.

File: app/components/graph/graph-service.js [hunk 1]
```diff
@@ -1183,7 +1197,7 @@ angular.module('lizard-nxt')
     var width = Graph.prototype._getWidth(dimensions),
     range = {min: 0, max: width},
     // Axis should run from zero to 100%
```
Comment (RoelvandenBerg) [inline app/components/graph/graph-service.js]: Axis should run from zero to 100% by default

**Comment by RoelvandenBerg** [comment]
@gijs kan jij hier naar kijken?

**Comment by gijs** [comment]
Looks good so far.

Suggestions:

 - Put the mouseover text in a popover or tooltip. This saves some vertical space and looks less littered.
screen shot 2016-11-30 at 14 44 40

**Comment by RoelvandenBerg** [comment]
En nu met tooltip.

**Pull request merged**